client/my-sites/email/email-management/titan-manage-mailboxes/index.jsx

```diff
@@ -56,12 +61,81 @@ class TitanManageMailboxes extends Component {
 		translate: PropTypes.func.isRequired,
 	};
 
+	state = { isDesktop: isDesktop() };
+	unsubscribe = () => {};
+
+	componentDidMount() {
+		this.unsubscribe = subscribeIsDesktop( ( isActive ) => {
+			this.setState( { isDesktop: isActive } );
+		} );
+	}
+
+	componentWillUnmount() {
+		this.unsubscribe();
+	}
+
 	handleBack = () => {
 		const { currentRoute, selectedDomainName, selectedSite } = this.props;
 
 		page( emailManagement( selectedSite.slug, selectedDomainName, currentRoute ) );
 	};
 
+	buildNavigationItem = ( { context, description, isDisabled = false, materialIcon, text } ) => ( {
+		description,
+		disabled: isDisabled,
+		materialIcon,
+		path: this.getPath( context ),
+		text,
+	} );
+
+	getManagedItems = () => {
```

Comment:
Name feels weird to me. `getMenuItems` or even `getNavigationItems` as they are being mapped at _VerticalNavItems_ and even they are being build with `buildNavigationItem`?

For consistency I think the best option is to call it `getNavigationItems` as this function internally uses the `buildNavigationItem`

Reply:
Fair